### DISCUSSION TITLE
文章发布时间和md文件中的时间不对

### DISCUSSION THREAD

**Discussion by mcyydscc** [hexojs/hexo#5073] [Question / Answer] [answered]
我的hexo是部署在vercel上的，我在md文件中设置了data，但是文章显示一直是他的重新部署时间
![image](https://user-images.githubusercontent.com/90653809/191886192-cdd8493a-c4bf-4452-85c7-a1feee5d2efe.png)
![image](https://user-images.githubusercontent.com/90653809/191886293-48cd0f7c-6683-4191-b5c8-b7f85571eb2b.png)

**Comment by stevenjoezhang** [comment, marked as answer]
日期是 `date`

**Reply by mcyydscc** [reply]
感谢，我复制错了
